Package.swift

```diff
@@ -64,7 +64,7 @@ let package = Package(
         .package(url: "https://github.com/apple/swift-argument-parser.git", .upToNextMajor(from: "0.4.3")),
         .package(url: "https://github.com/marmelroy/Zip.git", .upToNextMinor(from: "2.1.1")),
         .package(url: "https://github.com/tuist/GraphViz.git", .branch("tuist")),
-        .package(url: "https://github.com/fortmarek/SwiftGen", .branch("stable")),
+        .package(url: "https://github.com/hisaac/SwiftGen", .branch("add-symbolset-support")),
```

Comment:
What prevents us from pushing those changes upstream? Depending on a branch of a fork doesn't seem a good idea.

Reply:
I opened this pull request just to run CI with the changes that are on my fork. The intent is to to get the changes merged into @fortmarek's fork eventually, but I wanted to test the changes within Tuist before we do that.